assembl/static2/js/app/components/debate/common/postsFilter/filtersCategoryLabel.jsx

```diff
@@ -0,0 +1,28 @@
+// @flow
+import * as React from 'react';
+import { Translate } from 'react-redux-i18n';
+
+type Props = {
+  labelMsgId: string
+};
+
+class FiltersCategoryLabel extends React.Component<Props> {
```

Comment:
Please change this to a functional component, as you're not using state and other hooks, no class is needed.

Reply:
```
const FiltersCategoryLabel = ({labelMsgId}: Props) => {
   const categoryLabelStyle = {
      fontWeight: 'bold',
      textAlign: 'left',
      fontSize: '14px',
      padding: '20px 0 5px 20px',
      color: '#777'
    };
    return (
      <div style={categoryLabelStyle}>
        <strong>
          <Translate value={labelMsgId} />
        </strong>
      </div>
    );
  }
};
```

Reply:
Ok, tried it, I can't, there are too many embedded classes at different places that conflict with my css if I put it in a separate file, so I will close this PR for now and I will come back to it when the css refacto is done